core/domain/translatable_object_registry.py

```diff
@@ -60,7 +78,10 @@ def get_all_class_names(cls):
         return sorted(cls._translatable_objects_dict.keys())
 
     @classmethod
-    def get_object_class(cls, obj_type):
+    def get_object_class(
+        cls,
+        obj_type: TranslatableObjectNames
+    ) -> TranslatableObjectClasses:
```

Comment:
Shoudl you use `@override` here to specify which class is returned depending on the key?

Reply:
yes , we can absolutely do that . Also since we have a handful of keys and values ( class name and classes ) we can use @override here . Let me mend it real quick